decidim-meetings/app/controllers/decidim/meetings/admin/meeting_closes_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+module Decidim
+  module Meetings
+    module Admin
+      # This controller allows an admin to manage meetings from a Participatory Process
+      class MeetingClosesController < Admin::ApplicationController
+        helper ResourceHelper
+
+        def edit
```

Comment:
Yeah, I know there's no model for this. See this controller:

https://github.com/AjuntamentdeBarcelona/decidim/blob/master/decidim-admin/app/controllers/decidim/admin/participatory_process_step_activations_controller.rb

Activating a step is not related to an "activation model" either, but I'm creating the activation, that's why I used create/destroy there (destroy was removed since we decided that we could not deactivate steps anymore).

Anyway, that might just be me here, might be just my opinion.